sign/rpmgensig.c

```diff
@@ -168,6 +169,12 @@ static rpmtd makeSigTag(Header sigh, int ishdr, uint8_t *pkt, size_t pktlen)
 	break;
     }
 
+    ver = pgpDigParamsVersion(sigp);
+    if (ver < 4) {
+	rpmlog(RPMLOG_WARNING, _("Deprecated PGP signature version %d\n"),
```

Comment:
s/PGP/OpenPGP/

Reply:
Yup. I only made it that way because the surrounding messages also just talk about PGP, but maybe I should take the opportunity fix those instead...